lib/Doctrine/ORM/Tools/Console/Command/ClearCache/CollectionRegionCommand.php

```diff
@@ -82,6 +82,8 @@ protected function configure()
 
     /**
      * {@inheritdoc}
+     *
+     * @return int
```

Comment:
Why is this needed for Symfony commands? This return type is not inherited from a native type like `Countable` or `ArrayIterator`, I thought this was only about native types. Isn't it?

Reply:
When used with `DebugClassLoader`, this saves a deprecation: `@return` means the same as `#[ReturnTypeWillChange]` to it.

Reply:
Ok, I thought `DebugClassLoader` wouldn't need it in the first place, since no stubs are involved, only normal PHP files. Apparently that's not the case.